db/warehouse/migrate/20241122135153_add_household_move_in_date_to_apr_client.rb

```diff
@@ -0,0 +1,6 @@
+class AddHouseholdMoveInDateToAprClient < ActiveRecord::Migration[7.0]
+  def change
+    add_column :hud_report_apr_clients, :hoh_move_in_date, :date
+    add_column :hud_report_apr_clients, :household_move_in_date, :date
```

Comment:
What's the difference?  Looking below, I think `household_move_in_date` would be the client's move-in date if they have one, or the same as the `hoh_move_in_date` if they don't?  If that's the case, maybe `household_move_in_date` should be `move_in_date`?

Reply:
Maybe more to the point, I think we've had some clarity on move-in dates in the past few months.  Overall it should be pulled from the HoH unless the client entered after the HoH move-in date, in which case it should be the `EntryDate` from the client in question.  
Do you think pivoting to that would keep tests passing?

Reply:
This one came from the universe description for 22f in https://files.hudexchange.info/resources/documents/FY24-HMIS-Programming-Specifications-CoC-APR-and-ESG-CAPER.pdf

> Universe: All active clients where the head of household had a move-in date in the report date range plus leavers who exited in the date range and
never had a move-in date.

I believe I originally had it was originally just using the `move_in_date` and switching it to use the `household_move_in_date` field didn't quite get it right, but I can definitely run it through some more testing to see if theres a way to finesse it some more.

Reply:
Was that supposed to be `[housing move-in date]`?

Reply:
> What's the difference? Looking below, I think `household_move_in_date` would be the client's move-in date if they have one, or the same as the `hoh_move_in_date` if they don't? If that's the case, maybe `household_move_in_date` should be `move_in_date`?

The `household_move_in_date` is pulling from the HoH unless the client and HoH entry dates don't match, the `hoh_move_in_date` is specifically the move in date for the HoH in the enrollment.